resources/assets/js/views/FSL/cash_logger/cash_logger.vue

```diff
@@ -27,189 +27,207 @@
                             href="#payment"
                             @click.prevent="setActive('payments')"
                             role="tab"
-                            >
+                        >
                             View Payments
                         </a>
                     </li>
                 </ul>
             </div>
             <div class="tab-content py-3">
-            <div class="tab-pane fade" :class="{ 'active show': isActive('home') }" id="home">
-                <AutoCompleteSearch
-                    title=""
-                    @customer-selected="processForm"
-                    :url="'/api/customer/autocomplete'" />
-
-                <transition name="fade">
-                    <div v-if="customer && show">
-                        <customer-profile :view-customer="customer"/>
-                        <custom-header :title="'All order(s)'"/>
-
-                        <div class="mt-5 mb-3 attendance-head">
-                            <div class="row px-4 pt-3 pb-4 text-center">
-                                <div class="col light-heading" style="max-width: 100px">S/No.</div>
-                                <div class="col light-heading" v-for="header in headers">{{header}}</div>
+                <div class="tab-pane fade" :class="{ 'active show': isActive('home') }" id="home">
+                    <AutoCompleteSearch
+                        title=""
+                        @customer-selected="processForm"
+                        :url="'/api/customer/autocomplete'"/>
+
+                    <transition name="fade">
+                        <div v-if="customer && show">
+                            <customer-profile :view-customer="customer"/>
+                            <custom-header :title="'All order(s)'"/>
+
+                            <div class="mt-5 mb-3 attendance-head">
+                                <div class="row px-4 pt-3 pb-4 text-center">
+                                    <div class="col light-heading" style="max-width: 100px">S/No.</div>
+                                    <div class="col light-heading" v-for="header in headers">{{header}}</div>
+                                </div>
                             </div>
-                        </div>
 
-                        <div class="tab-content mt-1 attendance-body">
-                            <div class="tab-pane active text-center" v-if="show && customer.orders.length > 0">
-                                <div class="mb-3 row attendance-item" v-for="(order, index) in customer.orders">
-                                    <div class="col-12 col-xs-2 col-md col-lg d-flex align-items-center"
-                                         style="max-width: 100px">
-                                        <span class="user mx-auto">{{index+1}}</span>
-                                        <span v-if="$route.meta.customSMS">
+                            <div class="tab-content mt-1 attendance-body">
```

Comment:
Can this be re used in the third screen?
Please follow the mockup design
Also when you click on the comment  the full comment isn't revealed.